public/car.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="98" height="98" viewBox="0 0 98 98" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this svg has padding of sorts, which the washing / dryer icon does not have, which is slightly messing up the size of the icons on the page

Reply:
I think the easiest fix for this is to ask Kayln to update the padding around the SVG on Figma.